### PR TITLE
feat: init github to database

### LINKED ISSUE #95: Connect github-module to database

### PR DESCRIPTION
**Pull request by Cahllagerfeld**
closes #95 

- [x] actual functionality
- [x] unit-tests
- [x] e2e-tests

### CODE REVIEW

**Comment by Cahllagerfeld** [comment]
This PR is ready for review

File: src/github/github.service.ts [hunk 1]
```diff
@@ -112,23 +111,67 @@ export class GithubService {
...
+        ? this.mappingService.mapCommunityState(
+            body.event,
+            {} as CommunityStats,
+          )
+        : {},
+      followers: body.followers,
+      repos: body.repos,
+      blog: body.blog,
+      organization: body.organization,
+      createdOn: new Date(Date.now()),
```
Comment (eddiejaoude) [inline src/github/github.service.ts:165]: Could this be...

```suggestion
      createdOn: new Date(),
```

File: src/github/github.service.ts [hunk 2]
```diff
@@ -3,64 +3,58 @@ import { CommunityStats, GithubProfile } from './interfaces/github.interface';
...
 
-  findOne(id: number): GithubProfile {
-    const githubProfile = this.githubProfiles.find(
-      (profile) => profile.id === id,
+  findOne(id: string): Observable<GithubProfile> {
+    return this.astraService.get<GithubProfile>(id).pipe(
+      filter((data: GithubProfile) => {
+        if (data === null) {
+          throw new HttpException(
+            `no standup for ${id} found`,
```
Comment (eddiejaoude) [inline src/github/github.service.ts:48]: Should `standup` be something else?

**Comment by Cahllagerfeld** [comment]
Adjusted the reviewed issues

**Review by eddiejaoude — APPROVED** [review]
LGTM 👍

**Pull request merged**